code/__DEFINES/nitrile.dm

```diff
@@ -0,0 +1,5 @@
+///normal use time for syringes, pills, patches and medigels
+#define NORMAL_USE 3 SECONDS
```

Comment:
These are bad names (should be scoped to nitrile gloves, i.e. `NORMAL_USE_NO_NITRILES`), and it's bad that this is inside a file called "nitrile.dm", when it is for syringes, pills, patches, and medigels.

I think it would be better if instead of this, you just had a define for a multiplier to use--`#define NITRILE_GLOVE_SPEED_MULTIPLIER 0.5`